snscrape/modules/twitter.py

```diff
@@ -845,9 +922,19 @@ def _cli_from_args(cls, args):
 
 
 class TwitterTweetScraper(_TwitterAPIScraper):
+	'''Scraper object designed to scrape a specific tweet or thread surrounding it.'''
+
 	name = 'twitter-tweet'
 
 	def __init__(self, tweetId, mode = TwitterTweetScraperMode.SINGLE, **kwargs):
+		'''
+		Args:
+			tweetId: ID of the tweet.
+			mode: Scraping mode. Defaults to TwitterTweetScraperMode.SINGLE.
+		Yields:
+			Individual tweet.
```

Comment:
Correct. It may yield zero tweets as well if the requested tweet was deleted or the ID is invalid, for example.

But also, the 'Yields' part doesn't belong on the `__init__` method at all. It's `get_items` which yields the `Item`s (as correctly documented in other parts of the file).

Reply:
Yep, that confused me too at first. Originally I thought the `super.__init__` call automatically used the get_items scraper :sweat_smile:

Reply:
Whoops, it slipped through. Thanks for pointing this out! Will rectify.